packages/account-v2/src/modules/package.json

```diff
@@ -0,0 +1,84 @@
+{
+  "name": "@deriv-lib/account-v2-lib",
+  "version": "1.0.0",
+  "main": "dist/js/index.js",
+  "sideEffects": ["*.css"],
+  "types": "dist/types/index.d.ts",
+  "engines": {
+    "node": "18.x"
+  },
+  "scripts": {
+    "prebuild": "node scripts/webpack-entries.js",
+    "build": "NODE_ENV=production rimraf dist && NODE_OPTIONS='-r ts-node/register' webpack --progress --config \"./webpack.config.ts\"",
+    "prepublishOnly": "npm run build"
+  },
+  "dependencies": {
+    "class-variance-authority": "^0.7.0",
+    "clsx": "^2.0.0",
+    "i18n-iso-countries": "^6.8.0",
+    "i18next": "^22.4.6",
+    "usehooks-ts": "^2.7.0"
+  },
+  "files": [
+    "dist",
+    "README.md"
+  ],
+  "peerDependencies": {
+    "@deriv-com/ui": "1.11.2",
+    "@deriv/quill-icons": "^1.18.3",
+    "@deriv/api-v2": "1.0.0",
+    "yup": "^0.32.11",
+    "formik": "^2.1.4",
+    "react-calendar": "^4.7.0",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-dropzone": "11.0.1",
+    "react-i18next": "^11.11.0",
+    "react-router-dom": "^5.2.0",
+    "tailwind-merge": "^1.14.0"
+  },
+  "devDependencies": {
+    "@deriv-com/ui": "1.11.2",
+    "@deriv/api-v2": "1.0.0",
```

Comment:
They are in both places peerDependencies and devDependencies

Reply:
Updated.